### DISCUSSION TITLE
Connecting to database via windows integrated security

### DISCUSSION THREAD

**Discussion by mmilardic** [prisma/prisma#23614] [Q&A]
### Question

I'm trying to connect to our database with the following options:
```
datasource db {
  provider = "sqlserver"
  url      = "sqlserver://serverName:port;integratedSecurity=true;trustServerCertificate=true;encrypt=true;"
}
```

I'm getting:
`Error: Schema engine error:
Performing a TLS handshake
Trusting the server certificate without validation.
TLS handshake successful
Login failed for user ''.
[libs/user-facing-errors/src/quaint.rs:125:18] internal error: entered unreachable code
`

It is a SQL Server 15 (2019)

I'm on macOs and I have a Kerberos ticket.

I have no problems connecting to the DB server via Azure Data Explorer and TypeOrm .

TypeOrm using the following options:
` type: 'mssql',
  host: db.host,
  port: db.port,
  database: db.database,
  authentication: {
      type: 'ntlm',
      options: {
          userName: db.username,
          password: db.password,
          domain: db.domain,
      },
  },`

### How to reproduce (optional)

1. Create schema.prisma with the minimum requirements
2. Run: `npx prisma generate dev`
3. See error:
`Error: Schema engine error:
Performing a TLS handshake
Trusting the server certificate without validation.
TLS handshake successful
Login failed for user ''.
[libs/user-facing-errors/src/quaint.rs:125:18] internal error: entered unreachable code
`

### Expected behavior (optional)

Should connect to the DB

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
`generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "sqlserver"
  url      = "sqlserver://serverName:port;integratedSecurity=true;trustServerCertificate=true;encrypt=true;"
}`
```

- OS: macOS
- Database: SQL Server
- Node.js version: 20.9.0
- Prisma version: 5.11.0

**Comment by ludralph** [comment]
Hi @mmilardic 👋 

Can you adjust your connection string to follow the format specified in the [documentation](https://www.prisma.io/docs/orm/overview/databases/sql-server#connection-details)?

**Reply by mmilardic** [reply]
Hey @ludralph, I must connect to the DB using the windows integrated security, as documented [here](https://www.prisma.io/docs/orm/overview/databases/sql-server#using-integrated-security-windows-only).

But yes, I've tried what you linked, it gives me the same error.

**Reply by ludralph** [reply]
Hi @mmilardic 

Did you find a way to resolve this issue? If no, can you try the suggestion from this previous [discussion](https://github.com/prisma/prisma/discussions/5817#discussioncomment-401691)?

**Reply by mmilardic** [reply]
Hey @ludralph , I don't remember if I've tried the solution from the discussion you linked. 

Time was of essence so I switched to typeOrm and made it work. Appreciate you guys revisiting this :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
